heim-runtime/tests/linux.rs

```diff
@@ -0,0 +1,41 @@
+#[cfg(target_os = "linux")]
```

Comment:
You can just use `#![cfg(target_os = "linux")]` on top of the module instead of wrapping each line with `#[cfg]`